### ISSUE TITLE
Esconder emblemas.

### ISSUE THREAD

**Issue by RuffyTheLindow** [MrPowerGamerBR/owo#353]
**Qual é a sua sugestão?**

Uma função ou no site da Loritta ou nos comandos da mesma para esconder emblemas.

**Porque a sua sugestão deve ser adicionada na Loritta?**

Tem várias pessoas que tem emblemas que não desejam no próprio perfil, ou até tem o perfil lotado delas, e não quer sair dos servidores que tem elas, inclusive, sou uma delas, e é por isso que eu pedi isto.

**Comment by MrPowerGamerBR** [comment]
Sim, seria legal, o problema é que teria que mudar como as badges funcionam para que cada uma tivesse um ID, mas não seria impossível
